src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -109,6 +110,11 @@
     /** The maximum number of warnings that are tolerated. */
     private int maxWarnings = Integer.MAX_VALUE;
 
+    /** The checker threads number. */
+    private int checkerThreadsNumber = 1;
+    /** The tree walker threads number. */
+    private int treeWalkerThreadsNumber = 1;
```

Comment:
@sabaka , why we are updating ANT launcher now ?
do you plan to force Travis CI to run build in MT mode always to do self testing of MT on each commit ? if yes, it might be reasonable to do this when we are ready for this.